.travis.yml

```diff
@@ -4,16 +4,26 @@ go:
   - 1.8
   - tip
 
+addons:
+  apt:
+    packages:
+      - libonig-dev
+
 matrix:
   allow_failures:
     - go: tip
   fast_finish: true
 
+env:
```

Comment:
Please rebase to master since with the publication of `enry-java`, it has changed quite a lot.